service-base/src/main/java/org/eclipse/hono/service/plan/PrometheusBasedResourceLimitChecksProperties.java

```diff
@@ -0,0 +1,60 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.hono.service.plan;
+
+/**
+ * A class for managing configuration properties for PrometheusBasedResourceLimitChecks.
+ *
+ */
+public class PrometheusBasedResourceLimitChecksProperties {
```

Comment:
if these properties are specific to `PrometheusBasedResourceLimitChecks` then I do not see any reason why we should keep these properties in a separate class and not simply add them to `PrometheusBasedResourceLimitChecks` itself.
If these properties are generic and are relevant for all types of `ResourceLImitChecks` then this class should be named generically, wouldn't you agree?
Personally, I believe that the former is true, though ...

Reply:
@sophokles73 Do you mean directly into `PrometheusBasedResourceLimitChecks` as below?
```
    @Value(value = "${hono.plan.prometheus.host}")
    private String host;
    @Value(value = "${hono.plan.prometheus.port}")
    private String port;
```

Reply:
No, by moving all the properties along with the getters and setters to `PrometheusBasedResourceLimitChecks` and then simply move the `@ConfigurationProperties(prefix = "hono.plan.prometheusBased")` annotation from the  *resourceLimitChecksProperties* method to the *resourceLimitChecks* method in `AbstractAdapterConfig` (and, of course, removing the former method from the configuration) ...